pkg/backup/backup.go

```diff
@@ -28,6 +28,8 @@ import (
 	"path/filepath"
 	"time"
 
+	corev1api "k8s.io/api/core/v1"
```

Comment:
why not add it in second section, along `k8s.io/apimachinery` packages?

Reply:
re-positioned the import, I think make update had added it there.